### DISCUSSION TITLE
Please allow removing this notification that happens when inference is stopped/restarted each time

### DISCUSSION THREAD

**Discussion by cleverestx** [VoltaML/voltaML-fast-stable-diffusion#91] [Ideas]
For single users, not really useful and blocks too much...thank you!

![image](https://github.com/VoltaML/voltaML-fast-stable-diffusion/assets/28905253/263fee56-6c60-4dd0-8496-410a0934292f)

Thank you!

**Comment by Stax124** [comment]
Patched in dd42c25, it won't pop up if the Continuous Generation is set to something else than 0
